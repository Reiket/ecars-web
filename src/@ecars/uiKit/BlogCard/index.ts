import {BlogCardContent} from '@ecars/uiKit/BlogCard/components/BlogCardContent';
import {BlogCardComponent} from '@ecars/uiKit/BlogCard/BlogCardComponent';
import {BlogCardImage} from '@ecars/uiKit/BlogCard/components/BlogCardImage';

export const BlogCard = Object.assign(BlogCardComponent, {
  Content: BlogCardContent,
  Image: BlogCardImage,
});
