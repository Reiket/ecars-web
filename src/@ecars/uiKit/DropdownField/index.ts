import {DropdownFieldComponent} from '@ecars/uiKit/DropdownField/DropdownFieldComponent';
import {DropdownFieldWrapper} from '@ecars/uiKit/DropdownField/components/DropdownFieldWrapper';

export const DropdownField = Object.assign(DropdownFieldComponent, {
  Wrapper: DropdownFieldWrapper,
});
