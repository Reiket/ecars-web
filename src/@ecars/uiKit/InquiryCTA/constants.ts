import {withBlockClass} from 'ecars-web-lib';
import {PageWrapper} from '@ecars/uiKit/Page/components/PageWrapper';

export const InquiryCtaWrapperHOC = withBlockClass(PageWrapper, 'inquiry-cta');
