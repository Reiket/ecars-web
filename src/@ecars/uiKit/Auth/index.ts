import {AuthLayout} from '@ecars/uiKit/Auth/components/AuthLayout';
import {AuthPageWrapperHOC} from '@ecars/uiKit/Auth/constants';

export const AuthPage = Object.assign(AuthLayout, {
  Wrapper: AuthPageWrapperHOC,
});