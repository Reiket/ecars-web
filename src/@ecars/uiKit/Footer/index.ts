import {FooterComponent} from '@ecars/uiKit/Footer/FooterComponent';
import {FooterCopyWrapperHOC, FooterWrapperHOC} from '@ecars/uiKit/Footer/constants';
import {FooterCopy} from '@ecars/uiKit/Footer/components/FooterCopy';
import {FooterCopyList} from '@ecars/uiKit/Footer/components/FooterCopyList';
import {FooterColumn} from '@ecars/uiKit/Footer/components/FooterColumn';
import {FooterLogoBlock} from '@ecars/uiKit/Footer/components/FooterLogoBlock';
import {FooterLinkList} from '@ecars/uiKit/Footer/components/FooterLinkList';

export const Footer = Object.assign(FooterComponent, {
  Wrapper: FooterWrapperHOC,
  Column: FooterColumn,
  LogoBlock: FooterLogoBlock,
  LinkList: FooterLinkList,
});

export const Copy = Object.assign(FooterCopy, {
  Wrapper: FooterCopyWrapperHOC,
  List: FooterCopyList,
});
