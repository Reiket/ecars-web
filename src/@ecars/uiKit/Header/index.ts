import {HeaderComponent} from '@ecars/uiKit/Header/HeaderComponent';
import {HeaderNavbarWrapperHOC, HeaderWrapperHOC} from '@ecars/uiKit/Header/constants';
import {HeaderNavbar} from '@ecars/uiKit/Header/components/HeaderNavbar';
import {HeaderUtils} from '@ecars/uiKit/Header/components/HeaderUtils';
import {HeaderBurger} from '@ecars/uiKit/Header/components/HeaderBurger';
import {HeaderNavbarList} from '@ecars/uiKit/Header/components/HeaderNavbarList';

export const Header = Object.assign(HeaderComponent, {
  Wrapper: HeaderWrapperHOC,
  Utils: HeaderUtils,
  Burger: HeaderBurger,
});

export const Navbar = Object.assign(HeaderNavbar, {
  Wrapper: HeaderNavbarWrapperHOC,
  List: HeaderNavbarList,
});
