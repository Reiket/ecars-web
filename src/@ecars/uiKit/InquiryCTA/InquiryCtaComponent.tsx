import type {FC} from 'react';
import {InquiryCTA} from '@ecars/uiKit/InquiryCTA';
import {Button} from 'ecars-web-lib';

export const InquiryCtaComponent: FC = () => (
  <InquiryCTA.Wrapper
    name="inquiry-cta"
    size="sm"
  >
    <div className="inquiry-cta__body">
      <div className="inquiry-cta__block">
        <h2 className="inquiry-cta__title block-title">
          Send an inquiry now and get the best offers suitable for your requirements
        </h2>
        <p className="inquiry-cta__text">
          Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim
          velit mollit. Exercitation veniam consequat sunt nostrud amet.
        </p>
      </div>
      <Button
        size="big"
        color="green"
      >
        Get a quote
      </Button>
    </div>
  </InquiryCTA.Wrapper>
);
