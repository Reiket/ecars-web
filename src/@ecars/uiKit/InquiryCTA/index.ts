import {InquiryCtaComponent} from '@ecars/uiKit/InquiryCTA/InquiryCtaComponent';
import {InquiryCtaWrapperHOC} from '@ecars/uiKit/InquiryCTA/constants';

export const InquiryCTA = Object.assign(InquiryCtaComponent, {
  Wrapper: InquiryCtaWrapperHOC,
});
