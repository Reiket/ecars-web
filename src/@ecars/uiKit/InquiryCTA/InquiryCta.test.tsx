import {render} from '@testing-library/react';
import {InquiryCTA} from '@ecars/uiKit/InquiryCTA';

describe('InquiryCta Component', () => {
  test('render component correctly', () => {
    const {container} = render(<InquiryCTA />);
    expect(container).toMatchSnapshot();
    expect(container).toBeInTheDocument();
  });
});
