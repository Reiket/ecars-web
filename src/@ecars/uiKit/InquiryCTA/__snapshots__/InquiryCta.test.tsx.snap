// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`InquiryCta Component > render component correctly 1`] = `
<div>
  <div
    class="inquiry-cta"
  >
    <section
      class="inquiry-cta__section section"
    >
      <div
        class="inquiry-cta__container container container--sm"
      >
        <div
          class="inquiry-cta__body"
        >
          <div
            class="inquiry-cta__block"
          >
            <h2
              class="inquiry-cta__title block-title"
            >
              Send an inquiry now and get the best offers suitable for your requirements
            </h2>
            <p
              class="inquiry-cta__text"
            >
              Amet minim mollit non deserunt ullamco est sit aliqua dolor do amet sint. Velit officia consequat duis enim velit mollit. Exercitation veniam consequat sunt nostrud amet.
            </p>
          </div>
          <button
            class="inquiry-cta__button button button--big button--green"
            data-testid="buttonTestId"
            type="button"
          >
            Get a quote
          </button>
        </div>
      </div>
    </section>
  </div>
</div>
`;
